### PR TITLE
Document permissions api

### PR DESCRIPTION
**Pull request by artandor**
## Proposed Changes

This PR try to answer the issue exposed in #774 
I documented the permissions required for every API route in order to have a clear overview of the required permissions when checking the documentation.

## Types of Changes

What types of changes does your code introduce to this project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes issue #NNNN)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after
creating the PR. If you're unsure about any of them, don't hesitate to
ask on the mailing list. We're here to help! This is simply a reminder
of what we are going to look for before merging your code._

- [x] I have read the `CONTRIBUTING.md` document
- [x] I have signed the CA (see https://cla.pivotal.io/sign/rabbitmq)
- [x] All tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in related repositories

### CODE REVIEW

File: priv/www/api/index.html [hunk 1]
```diff
@@ -403,6 +443,10 @@ <h2>Reference</h2>
         <td>X</td>
         <td></td>
         <td class="path">/api/exchanges/<i>vhost</i>/<i>name</i></td>
+        <td>management</td>
+        <td>User added to vhost<br/>
```
Comment (michaelklishin) [inline priv/www/api/index.html:447]: A user cannot be "added to a vhost". A user can have no or some permissions in a vhost.

File: priv/www/api/index.html [hunk 2]
```diff
@@ -403,6 +443,10 @@ <h2>Reference</h2>
...
+          [PUT & DELETE] Configuration
```
Comment (michaelklishin) [inline priv/www/api/index.html:448]: "Configuration" is not a role or permission in RabbitMQ. `configure` is a permission (for a single virtual host).

File: priv/www/api/index.html [hunk 3]
```diff
@@ -437,6 +485,8 @@ <h2>Reference</h2>
         <td></td>
         <td>X</td>
         <td class="path">/api/exchanges/<i>vhost</i>/<i>name</i>/publish</td>
+        <td>management</td>
+        <td>User added to vhost & write permission</td>
```
Comment (michaelklishin) [inline priv/www/api/index.html:489]: This should be clear as to what the `write` permission is on: the target exchange.

**Review by michaelklishin — CHANGES_REQUESTED** [review]
Thank you for your time. This needs some changes before it would be considered. Specifically:

 * Nearly all whitespace changes are not relevant and it would be great to undo them
 * "A user added to a vhost" is neither specific nor a correct term. A user can have some permissions in a vhost. In which case we should specify what they are.
 * "Administrator or policymaker" should be just "policymaker" since "administrator" includes "policymaker" just like "policymaker" is "management" plus the ability to operate on policies
 * Please changes `[PUT & DELETE]` to `PUT`, `DELETE`
 * It may be worth having "Tags: …" and "object permissions: …" instead of "administrator or policymaker & added to vhost" (free form in a single line, plus not specific enough)

**Comment by dcorbacho** [comment]
@artandor The management plugin has been moved to the [server](https://github.com/rabbitmq/rabbitmq-server) repository. Please re-submit your PR against this repository once you have applied the requested changes.

Many thanks for your contribution!

**Pull request closed without merging**